vsgenstubs/_vapoursynth.part.pyi

```diff
@@ -415,4 +417,6 @@ def get_core(threads: typing.Optional[int]=None, add_cache: typing.Optional[bool
 
 class _CoreProxy(Core):
     core: Core
+
```

Comment:
I had omitted the white-spaces to denote that the belong together.

Reply:
I was just keeping it consistent with how the other classes looked, like with ColorFamily or PresetFormat.

Reply:
I see.